pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/sort/Alias.java

```diff
@@ -32,19 +34,21 @@
     private String value;
 
     public static Alias of(SortBy sortBy) {
-        Alias alias;
-        try {
-            String tableAlias;
-            if (ESObjectConstants.DATA.equals(sortBy.getTableAlias())) {
-                tableAlias = StringUtils.isNotBlank(sortBy.getOuterProperty()) ? sortBy.getOuterProperty() : sortBy.getInnerProperty();
-            } else {
-                tableAlias = sortBy.getTableAlias();
-            }
-            alias = valueOf(tableAlias.toUpperCase());
-        } catch (IllegalArgumentException iae) {
-            alias = ACTIVITIES;
+        String tableAlias;
+        if (ESObjectConstants.DATA.equals(sortBy.getTableAlias())) {
+            tableAlias = StringUtils.isNotBlank(sortBy.getOuterProperty()) ? sortBy.getOuterProperty() : sortBy.getInnerProperty();
+        } else {
+            tableAlias = sortBy.getTableAlias();
         }
-        return alias;
+        return of(tableAlias);
+    }
+
+    public static Alias of(String tableAlias) {
+        return Enums.getIfPresent(Alias.class, tableAlias.toUpperCase()).or(ACTIVITIES);
```

Comment:
I don't see the point of adding some 3rd party library complexity here, I believe `try/catch` was doing its job properly and is easily understandable special-case pattern.